docs/tutorial/using-checkbox/checkbox-commands.rst

```diff
@@ -0,0 +1,296 @@
+    test plan 'com.canonical.certification::acpi-automated'
+    test plan 'com.canonical.certification::audio-cert-full'
+    (...)
+    test plan 'com.canonical.certification::tpm-cert-focal-automated'
+    test plan 'com.canonical.certification::tpm-cert-automated'
+
+Combined with a ``grep`` command, it is possible to find all the test plans
+related to audio testing:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list "test plan" | grep audio
+
+    test plan 'com.canonical.certification::audio-cert-full'
+    test plan 'com.canonical.certification::audio-cert-manual'
+    (...)
+    test plan 'com.canonical.certification::after-suspend-audio-pa-manual'
+    test plan 'com.canonical.certification::after-suspend-audio-automated'
+
+Or let's list all the available jobs (test cases):
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list all-jobs
+    
+    id: com.canonical.certification::6lowpan/kconfig
+    kernel config options for 6LoWPAN
+    id: com.canonical.certification::IEEE_80211
+    Creates resource info for wifi supported protocols/interfaces
+    id: com.canonical.certification::acpi/oem_osi
+    test ACPI OEM _OSI strings
+    (...)
+
+By default, the output for the ``all-jobs`` option is to list the job
+identifier followed by its summary (or ``<missing summary>`` if there is no
+summary). We can tailor the output using the ``--format`` parameter and all the
+fields available from the jobs. To see what fields are available, run:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list all-jobs --format ?
+    
+    Available fields are:
+    _description, _purpose, _siblings, _steps, _summary, _verification, after, category_id, command, depends, environ, estimated_duration, flags, full_id, id, imports, plugin, require, requires, template-engine, template-filter, template-resource, template-unit, unit, user
+
+.. note::
+
+    These fields are explained in the :ref:`job` page.
+
+.. note::
+
+    The underscore before some of the fields names simply means the content
+    of this field can be translated into another language.
+
+To create a table listing each job id and their summary, run:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli list all-jobs --format "{id:30} | {_summary}\n"
+
+    6lowpan/kconfig                | kernel config options for 6LoWPAN
+    IEEE_80211                     | Creates resource info for wifi supported protocols/interfaces
+    acpi/oem_osi                   | test ACPI OEM _OSI strings
+    acpi_sleep_attachment          | <missing _summary>
+    (...)
+    xinput                         | Creates resource info from xinput output.
+    zapper_capabilities            | Get Zapper's setup capabilities
+    collect-manifest               | Collect the hardware manifest (interactively)
+    manifest                       | Hardware Manifest
+
+.. note::
+
+    ``\n`` and ``\t`` in the formatting string are interpreted and replaced
```

Comment:
format does not actually use \t here, maybe just put it there `"{id:15}\t | {_summary}\n"` just to get the point across

Reply:
Fixed in [20d9258](https://github.com/canonical/checkbox/pull/635/commits/20d92587d4c1323c44666a647f24b991ccabe62d)